app/src/main/java/de/grobox/transportr/favorites/trips/SavedSearchesViewModel.java

```diff
@@ -42,7 +42,7 @@ protected SavedSearchesViewModel(TransportrApplication application, TransportNet
 		this.savedSearches = searchesRepository.getFavoriteTrips();
 	}
 
-	LiveData<List<FavoriteTripItem>> getFavoriteTrips() {
+	public LiveData<List<FavoriteTripItem>> getFavoriteTrips() {
```

Comment:
Why did you need to change the visibility here? Do you access this ViewModel outside of this package?

Reply:
Yes, in `DirectionsActivity`
where I use it like `viewModel.getFavoriteTrips().observe(...)`